### DISCUSSION TITLE
Forked repositories being skipped

### DISCUSSION THREAD

**Discussion by alexgg** [renovatebot/renovate#16172] [Request Help] [answered]
Hi,

Running the renovate github app on a github organization, I don't manage to configure forked repositories not to be skipped.

For example a forked public repository at https://github.com/balena-os/balena-rockchip-rk3288, which is configured in https://github.com/balena-os/balena-rockchip-rk3288/blob/master/.github/renovate.json with `"includeForks": true` is still being skipped with:

```
DEBUG: No dangling containers to remove
INFO: Repository started
{
  "renovateVersion": "32.89.2"
}
DEBUG: Using localDir: /mnt/renovate/gh/balena-os/balena-rockchip-rk3288
DEBUG: PackageFiles.clear() - Package files deleted
{
  "baseBranches": []
}
DEBUG: resetMemCache()
DEBUG: initRepo("balena-os/balena-rockchip-rk3288")
DEBUG: balena-os/balena-rockchip-rk3288 default branch = master
DEBUG: Using app token for git init
DEBUG: GitHub 404
{
  "url": "repos/balena-os/balena-rockchip-rk3288/contents/renovate.json"
}
INFO: Repository is a fork and not manually configured - skipping
DEBUG: Repository result: fork, status: disabled, enabled: false, onboarded: undefined
DEBUG: Repository timing splits (milliseconds)
{
  "splits": {},
  "total": 1110
}
INFO: Repository finished
{
  "durationMs": 1110
}
```

Not sure what goes on, but the 404 above might indicate it is not able to read the renovate configuration? Non-forked repositories are working fine.

Any ideas?

**Comment by viceice** [comment, marked as answer]
for this to work `renovate.json` needs to be in repo root

**Comment by alexgg** [comment]
Hi, thanks for the answer, I'll try that straight away.

Is the `includeForks` the only option that needs to be specified in a `renovate.json` in the root repository, or are there other options with this limitation?

Will it work if the `renovate.json` is in the repository root, but it only contains an extends?

**Reply by rarkins** [reply]
It must be a `renovate.json` and must have `"includeForks": true`. You would then add an `extends` to it as well.

Note that if you were using other filenames such as `renovate.json5` then they will be ignored once `renovate.json` is found, so you can't put the rest of config in a separate config file - only `renovate.json`

**Reply by alexgg** [reply]
thanks @rarkins @viceice  that works

**Comment by MichaelKunze** [comment]
For future reference... The name of the config option has changed:

`"forkProcessing":` "enabled"`

See: https://docs.renovatebot.com/configuration-options/#forkprocessing

**Reply by CanerKaraca23** [reply]
It's not working for me.

**Reply by JounQin** [reply]
@CanerKaraca23 As @rarkins [answered](https://github.com/renovatebot/renovate/discussions/16172#discussioncomment-2994352), the config must be named `renovate.json`...

**Reply by CanerKaraca23** [reply]
> @CanerKaraca23 As @rarkins [answered](https://github.com/renovatebot/renovate/discussions/16172#discussioncomment-2994352), the config must be named `renovate.json`...

I already have https://github.com/CanerKaraca23/KernelSU-Next/blob/next/renovate.json

**Reply by JounQin** [reply]
Check the branch setting at `https://developer.mend.io/github/<org>/<repo>/-/settings`

**Reply by CanerKaraca23** [reply]
> Check the branch setting at https://developer.mend.io/github///-/settings

Which setting?

**Reply by JounQin** [reply]
`https://developer.mend.io/github/<org>/<repo>/-/settings`

<img width="1249" alt="image" src="https://github.com/user-attachments/assets/08e7e2ec-3b9f-467e-9610-f250165742a1" />

**Reply by CanerKaraca23** [reply]
> `https://developer.mend.io/github/<org>/<repo>/-/settings`
> 
> <img alt="image" width="1249" src="https://private-user-images.githubusercontent.com/8336744/428361088-08e7e2ec-3b9f-467e-9610-f250165742a1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDMzMTM2NzIsIm5iZiI6MTc0MzMxMzM3MiwicGF0aCI6Ii84MzM2NzQ0LzQyODM2MTA4OC0wOGU3ZTJlYy0zYjlmLTQ2N2UtOTYxMC1mMjUwMTY1NzQyYTEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDMzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAzMzBUMDU0MjUyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NDMwOTcwZWMwMmZiNTA4MTNkODZiYWYwNGQwZWI2NzFjNTFhMDdhMmM1MWFjY2ZkMDNkNDk2ZTVhNzdiYzI0MyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.ojHthLHRM0xCux8aHIkmCmUd67Qp-Agt8_Zx7O_mo5w">

yes, it's selected, still disabled

**Reply by oanalavinia** [reply]
Hi! I am also facing the same issue. I have the [renovate.json](https://github.com/xwikisas/application-diagram/blob/master/renovate.json) file on root for my forked repository, and have followed what was proposed in this discussion and on the documentation for [forkProcessing](https://docs.renovatebot.com/configuration-options/#forkprocessing). Even with these, renovate doesn't seems enabled for this repository, while for other non-forked repositories in my organization everything works well. Do you have any hint for what to look into now? Maybe I missed something
Note that I am using Mend hosted Forking renovate app

Edit: Sorry for the noise, I found after that Forking renovate is not working on forked repositories.

**Comment by Goodmanisrael27** [comment]
❤️
